client/state/themes/test/actions.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import sinon from 'sinon';
+import { expect } from 'chai';
+import { Map } from 'immutable';
+
+/**
+ * Internal dependencies
+ */
+import {
+	THEME_ACTIVATE_REQUEST,
+	THEME_ACTIVATE_REQUEST_SUCCESS,
+	THEME_ACTIVATE_REQUEST_FAILURE,
+} from 'state/action-types';
+import {
+	themeActivation,
+	themeActivated,
+	themeActivationFailed,
+	themeActivationSuccess,
+	activateTheme,
+} from '../actions';
+import useNock from 'test/helpers/use-nock';
+
+describe( 'actions', () => {
+	const spy = sinon.spy();
+
+	beforeEach( () => {
+		spy.reset();
+	} );
+
+	describe( '#themeActivation()', () => {
+		it( 'should return an action object', () => {
+			const themeId = 'twentysixteen';
+			const siteId = 2211667;
+			const expected = {
+				type: THEME_ACTIVATE_REQUEST,
+				themeId,
+				siteId,
+			};
+
+			const action = themeActivation( themeId, siteId );
+			expect( action ).to.eql( expected );
+		} );
+	} );
+
+	describe( '#themeActivated()', () => {
+		it( 'should return an action object', () => {
+			const theme = { id: 'twentysixteen' };
+			const siteId = 2211667;
+			const expected = {
+				type: THEME_ACTIVATE_REQUEST_SUCCESS,
+				theme,
+				siteId,
+			};
+
+			const action = themeActivated( theme, siteId );
+			expect( action ).to.eql( expected );
+		} );
+	} );
+
+	describe( '#themeActivationFailed()', () => {
+		it( 'should return an action object', () => {
+			const themeId = 'twentysixteen';
+			const siteId = 2211667;
+			const error = { error: 'theme_not_found', message: 'The specified theme was not found' };
+			const expected = {
+				type: THEME_ACTIVATE_REQUEST_FAILURE,
+				themeId,
+				siteId,
+				error,
+			};
+
+			const action = themeActivationFailed( themeId, siteId, error );
+			expect( action ).to.eql( expected );
+		} );
+	} );
+
+	const trackingData = {
```

Comment:
Stuff like this should probably live in a `before()` function inside a `context()` that is shared by all the tests that rely on it.